tools/ci.sh

```diff
@@ -82,47 +82,58 @@ function ci_code_size_setup {
 
 function ci_code_size_build {
     # check the following ports for the change in their code size
-    PORTS_TO_CHECK=bmusxpdv
+    # Override the list by setting PORTS_TO_CHECK in the environment before invoking ci.
+    : ${PORTS_TO_CHECK:=bmusxpdv}
+    
     SUBMODULES="lib/asf4 lib/berkeley-db-1.xx lib/btstack lib/cyw43-driver lib/lwip lib/mbedtls lib/micropython-lib lib/nxp_driver lib/pico-sdk lib/stm32lib lib/tinyusb"
```

Comment:
should this be moved into the metrics script, like `metrics.py submodule  "$PORTS_TO_CHECK"`? (which would "make submodules" in each location called for)